CnCNet.LauncherStub.csproj

```diff
@@ -1,14 +1,14 @@
 <Project Sdk="Microsoft.NET.Sdk.WindowsDesktop">
   <PropertyGroup>
-    <TargetFramework>net471</TargetFramework>
-    <LangVersion>11.0</LangVersion>
+    <TargetFrameworks>net40; net45; net471</TargetFrameworks>
```

Comment:
Hmm is there any reason for all those old frameworks and not just net48?

Reply:
And if it relies on 4.0, is there a need to compile anything else?

Reply:
the launcher prompts users about missing frameworks. Win7 comes with net35 (due to the need of accessing registry in 32bit view, the launcher cannot be downgraded to that old), while Windows 10 only comes with net46.

Reply:
Is there any benefit in compiling multiple versions then? I think we could settle on one version that fits the most and leave it at that.

Reply:
.net 4.0 does have limitations, for example the launcher can not differ amd64 from arm64. This is brought by net45. So that's why I reserved other two frameworks versions in case we need to upgrade the framework for some reasons, for example if Qualcomm’s process be popular in laptops someday

Reply:
I think it would be net40 then since only advance users who want to run .net 8.0 clients in an arm64 machine needs net45 launcher

Reply:
Why not net45? I don't think net40 vs net45 would have impact on win7 users, maybe Win8/8.1 though?

Reply:
I don't care about win8 users. Just wondering if there are some win7 users that only have .net 4.0 installed?

Reply:
I know only that net40 came as an update to Win7. Also Win8 has net45 bundled.

Reply:
Either way, I am fine with either.

Reply:
I remembered wrong. arm64 was supported on net471. So we should leave net40 as the default which is already done in GitHub CI

Reply:
I am in favor of `<TargetFrameworks>net40; net471</TargetFrameworks>`  over `<TargetFramework>net40</TargetFramework>` in case someone needs it for ARM64. The GitHub CI has been configured to compile net40 only, providing binary for downloads in Release